sites/postcss-preset-env/blog/can-use-boolean-custom-media.md

```diff
@@ -28,8 +28,8 @@ Just remember to remove the `true` and `false` values before you ship your code!
 @custom-media --dark (prefers-color-scheme: dark);
 
 @media (--tablet) and (--dark) {
-  body {
-    color: cyan;
-  }
+	body {
+		color: cyan;
+	}
```

Comment:
There was some inconsistency between tabs and spaces.
I don't have a preference between either in this context, but I do like the consistent. Everything is tabs now, but I am happy to change them to spaces.

Reply:
I love tabs, that's what they're for!